components/projects/DeleteProjectModal.tsx

```diff
@@ -0,0 +1,67 @@
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { trpc } from "@/utils/trpc";
+import type { Project } from "@prisma/client";
+import { ArrowRight, TrashIcon } from "lucide-react";
+import { Button, Modal, Paragraph } from "@/components/theme";
+import { showToast } from "../theme/showToast";
+
+interface DeleteProjectModalProps {
+  project: Project;
+}
+
+const DeleteProjectModal = ({ project }: DeleteProjectModalProps) => {
+  const [loading, setLoading] = useState(false);
+
+  const router = useRouter();
+
+  const { mutate: projectDeleteMutation, isLoading } =
+    trpc.projects.delete.useMutation({
+      onSuccess: () => {
+        showToast({
+          type: "success",
+          title: "Project Deleted successfully",
+          subtitle: "",
+        });
+        router.push("/projects");
+      },
+      onError: (error) => {
+        showToast({
+          type: "error",
+          title: "Project Delete failed",
+          subtitle: error.message,
+        });
+        setLoading(false);
+      },
+    });
+
+  const submitForm = () => {
+    setLoading(true);
+    projectDeleteMutation({
+      project,
+    });
+  };
+
+  return (
+    <Modal
+      button={
+        <Button type="button" variant="danger-outline">
+          Delete this project
+        </Button>
+      }
+      title="Delete this project?"
+    >
+      <div className="flex flex-col items-center gap-3">
+        <Paragraph size="sm" className="mt-4 text-center text-sm font-light">
```

Comment:
Better to show a form to type project name and confirm delete otherwise LGTM 🚀

Reply:
It's makes sense. I'll do this